stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/MessageInputViewModel.kt

```diff
@@ -33,6 +34,9 @@ public class MessageInputViewModel @JvmOverloads constructor(
     public val editMessage: MutableLiveData<Message?> = MutableLiveData()
     public val repliedMessage: MediatorLiveData<Message?> = MediatorLiveData()
 
+    private val _isDirectMessage: MutableLiveData<Boolean> = MutableLiveData()
+    public val isDirectMessage: LiveData<Boolean> = _isDirectMessage
```

Comment:
The `isDirectMessage` value will never change during the ViewModel's life. Maybe simple `Boolean` property would be enough instead of `LiveData<Boolean>`? 🤔

Reply:
It would be so ugly to have a regular plain value exposed from a ViewModel though - I agree this is overkill, but it's consistent API at least